src/assets/scripts/client/InputController.js

```diff
@@ -281,14 +286,15 @@ export default class InputController {
                 this.selectAircraft(aircraftModel);
             }
 
-            position = [
-                UiController.px_to_km(position[0]),
-                UiController.px_to_km(position[1])
+            currentMousePosition = [
+                CanvasStageModel.translatePixelsToKilometers(currentMousePosition[0]),
+                CanvasStageModel.translatePixelsToKilometers(currentMousePosition[1])
             ];
 
-            position[0] = parseFloat(position[0].toFixed(2));
-            position[1] = parseFloat(position[1].toFixed(2));
-            this.input.positions += `[${position.join(',')}]`;
+            currentMousePosition[0] = parseFloat(currentMousePosition[0].toFixed(2));
+            currentMousePosition[1] = parseFloat(currentMousePosition[1].toFixed(2));
+            // FIXME: what the is this?!
+            this.input.positions += `[${currentMousePosition.join(',')}]`;
```

Comment:
I guess it's storing the positions clicked? No idea why; likely to be _extremely_ old.